README.md

```diff
@@ -311,14 +319,19 @@ This will result in Kafka Broker making a request to authorization server every
 Specify the following `oauth.*` properties:
 - `oauth.introspection.endpoint.uri` (e.g.: "https://localhost:8443/auth/realms/demo/protocol/openid-connect/token/introspect")
 - `oauth.valid.issuer.uri` (e.g.: "https://localhost:8443/auth/realms/demo" - only access tokens issued by this issuer will be accepted)
-- `oauth.client.id` (e.g.: "kafka" - this is the client configuration id for Kafka Broker)
+- `oauth.client.id` (e.g.: "kafka" - this is the OAuth2 client configuration id for Kafka Broker)
 - `oauth.client.secret` (e.g.: "kafka-secret")
  
 Introspection endpoint should be protected. The `oauth.client.id` and `oauth.client.secret` specify Kafka Broker credentials for authenticating to access the introspection endpoint. 
 
 Some authorization servers don't provide the `iss` claim. In that case you would not set `oauth.valid.issuer.uri`, and you would explicitly turn off issuer checking by setting the following option to `false`:
 - `oauth.check.issuer` (e.g.: "false")
 
+You can enforce audience check, which is an OAuth2 mechanism to limit access to tokens that are explicitly issued for use by your protected server.
```

Comment:
Same comment

(also why are we repeating this verbatim , can't we just refer the reader to the previous duscussion?)

Reply:
It's structured to have 'fast local JWT checking' chapter with all the config options, and a separate 'introspection based validation' chapter. Some options are shared by both. But if you look closely you'll see that there are some nuances and even the part about audience checking is slightly different for 'fast JWT checking' chapter. There's this sentence:

"When audience check is enabled the `oauth.client.id` has to be configured:
- `oauth.client.id` (e.g.: "kafka" - this is the OAuth2 client configuration id for Kafka Broker)" 

Which is not present in 'Introspection based validation' chapter, because `oauth.client.id` is already a required configuration there.

I prefer it this way as it's more user-friendly than having the user jump around and trying to figure out if something fits together with something else.